enhancements/ingress/haproxy-reload-interval.md

```diff
@@ -100,10 +105,10 @@ To expose the `ReloadInterval` in HAProxy, the environment variable `RELOAD_INTE
     (*appsv1.Deployment, error){
         ...
         reloadInterval := 5 * time.Second
-	    if ci.Spec.TuningOptions.ReloadInterval != nil && ci.Spec.TuningOptions.ReloadInterval.Duration >= 1*time.Second {
+	    if ci.Spec.TuningOptions.ReloadInterval != nil && ci.Spec.TuningOptions.ReloadInterval.Duration >= 1 * time.Second && ci.Spec.TuningOptions.ReloadInterval.Duration <= 720 * time.Hour {
 		    reloadInterval = ci.Spec.TuningOptions.ReloadInterval.Duration
 	    }
-	    env = append(env, corev1.EnvVar{Name: RouterReloadIntervalEnvName, Value: durationToHAProxyTimespec(reloadInterval)})
+	    env = append(env, corev1.EnvVar{Name: RouterReloadIntervalEnvName, Value: strconv.Itoa(int(reloadInterval.Seconds()))})
```

Comment:
Does this work?
```suggestion
	    env = append(env, corev1.EnvVar{Name: RouterReloadIntervalEnvName, Value: fmt.Sprintf("%v", reloadInterval.Seconds())})
```
That avoids truncating to an integer and formats as a decimal (e.g., "10500ms" gets formatted as "10.5" seconds), and I think `RELOAD_INTERVAL` will properly handle values like "10.5", based on <https://go.dev/play/p/4w625xB8suO>.

Reply:
In that case, I can just take out the `int()` and just have `strconv.Itoa(reloadInterval.Seconds())`. Should I still use `fmt.Sprintf("%v", reloadInterval.Seconds())` instead?

Reply:
Nevermind, `strconv.Itoa()` doesn't work on floats. I'll change my code to use `fmt.Sprintf("%v", reloadInterval.Seconds())`, thanks!